### DISCUSSION TITLE
Test vectors for sequential circuits

### DISCUSSION THREAD

**Discussion by fdila** [logisim-evolution/logisim-evolution#950] [Q&A]
Hi!
First of all we are using logisim-evolution for our Computer Architecture course and I'd like to thank you for your great work.
I was wondering if there is some way to use the test vectors to test circuits that require multiple clock cycles, from the documentation I can only see ways to use them for combinational logic.

**Comment by R3dst0ne** [comment]
Sadly, I can't help you, but there was a discussion(#598) opened a while back regarding this topping.
It might be a good idea to continue this conversation there.
